<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutFERAllLeft">
     <item>
      <layout class="QHBoxLayout" name="horizontalFERTopButtonsLeft">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutFERTopPushButtonsLeft">
         <item>
          <widget class="QPushButton" name="pushButtonUpdateFERLeft">
           <property name="maximumSize">
            <size>
             <width>85</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Waree</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="toolTip">
            <string>Update the list of Field Extraction Rules based on the org/credentials entered above. </string>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_34">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutContentRadioButtonsLeft">
           <item alignment="Qt::AlignVCenter">
            <widget class="QRadioButton" name="radioButtonFERsAll">
             <property name="font">
              <font>
               <family>Waree</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>Show All FERs</string>
             </property>
             <property name="text">
              <string>All</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="enabled">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupContentLeft</string>
             </attribute>
            </widget>
           </item>
           <item alignment="Qt::AlignVCenter">
            <widget class="QRadioButton" name="radioButtonFERsShowInLeftOnly">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <family>Waree</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>Show FERs those only exist in Left</string>
             </property>
             <property name="text">
              <string>In Left Only</string>
             </property>
             <property name="enabled">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupContentLeft</string>
             </attribute>
            </widget>
           </item>
           <item alignment="Qt::AlignVCenter">
            <widget class="QRadioButton" name="radioButtonFERsShowInRightOnly">
             <property name="font">
              <font>
               <family>Waree</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>Show FERs those only exist in Right</string>
             </property>
             <property name="text">
              <string>In Right Only</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupContentLeft</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonFERsShowDifferent">
             <property name="toolTip">
              <string>Show FERs those in both sides but differ in query</string>
             </property>
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Diffrent</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupContentLeft</string>
             </attribute>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_17">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="FERListWidgetLeft">
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutFERBottomButtonsLeft">
       <item>
        <widget class="QPushButton" name="pushButtonFERDeleteLeft">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Delete the selected Field Extraction Rules. Be careful!</string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonFERBackupLeft">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Backup the selected Field Extraction Rules to JSON files. One file will be written per rule. </string>
         </property>
         <property name="text">
          <string>Backup</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonFERRestoreLeft">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Restore Field Extraction Rule(s) from JSON file(s). </string>
         </property>
         <property name="text">
          <string>Restore</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonFERJSONLeft">
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>View FER(s) JSON</string>
         </property>
         <property name="text">
          <string>JSON</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_40">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutFERCenterButton">
     <item>
      <spacer name="verticalSpacer_19">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>80</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFERCopyLeftToRight">
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Copy the the select Field Extraction Rule(s) from the left org to the right org. </string>
       </property>
       <property name="text">
        <string>Copy -&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFERCopyRightToLeft">
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Copy the the select Field Extraction Rule(s) from the right org to the left org. </string>
       </property>
       <property name="text">
        <string>&lt;- Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_20">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFERFindReplaceCopyLeftToRight">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Coming Soon!</string>
       </property>
       <property name="text">
        <string>Find 
Replace-&gt;
Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFERFindReplaceCopyRightToLeft">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Coming Soon!</string>
       </property>
       <property name="text">
        <string>Find 
&lt;-Replace
Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_21">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutFERAllRight">
     <item>
      <layout class="QHBoxLayout" name="horizontalFERTopButtonsRight">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutFERTopPushButtonsRight">
         <item>
          <widget class="QPushButton" name="pushButtonUpdateFERRight">
           <property name="maximumSize">
            <size>
             <width>85</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Waree</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="toolTip">
            <string>Update the list of Field Extraction Rules based on the org/credentials entered above. </string>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_37">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="FERListWidgetRight">
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutFERBottomButtonsRight">
       <item>
        <widget class="QPushButton" name="pushButtonFERDeleteRight">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Delete the selected Field Extraction Rules. Be careful!</string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonFERBackupRight">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Backup the selected Field Extraction Rules to JSON files. One file will be written per rule. </string>
         </property>
         <property name="text">
          <string>Backup</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonFERRestoreRight">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Restore Field Extraction Rule(s) from JSON file(s). </string>
         </property>
         <property name="text">
          <string>Restore</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonFERJSONRight">
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>View FER(s) JSON</string>
         </property>
         <property name="text">
          <string>JSON</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_39">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
